lib/portlayer/storage/vsphere/export.go

```diff
@@ -130,13 +131,19 @@ func (i *ImageStore) Export(op trace.Operation, id, ancestor string, spec *archi
 	// this allows us to assume it's an image
 	r, err := i.NewDataSource(op, ancestor)
 	if err != nil {
+		op.Debugf("Unable to get datasource for ancester: %s", err)
```

Comment:
Based on the TV advertisement that I keep seeing, I'm sure http://www.ancestor.com/ will help you there :P